### PR TITLE
Added example for "Sparse Fieldsets"

### PR DESCRIPTION
**Pull request by ksimka**
Example demonstrating how `fields` param works.

### CODE REVIEW

**Comment by dgeb** [comment]
Thanks for this, @ksimka!

We may ask you to retarget this to a new top-level examples page we're planning in #503.

**Comment by tkellen** [comment]
@ksimka We've just added the page @dgeb mentioned--would you mind refactoring?

**Comment by ksimka** [comment]
Yes, but I'm not sure that I understand #670 correctly. The response in the example depends on it. Could you please clarify if we've already resolved the confusion between `include` and `fields`?

**Comment by tkellen** [comment]
@dgeb is about to submit a PR that puts this issue to bed. Let's revisit after?

**Comment by ksimka** [comment]
> Let's revisit after?

Ok, I'll refactor right after #670 is closed.

**Comment by tkellen** [comment]
Just closed via #688.

**Comment by ksimka** [comment]
Sorry, I feel stupid, but still not sure if I get it right )

If `include=a.b.c&fields[A]=d,e` where `a` consists of `A`, `b` — `B` and `c` — `C`, so `A` will not have linkage to objects `B` (that will be in `included`), and we can't link them properly. Or does that mean to not include `b` at all? Then `C` objects can't be linked to anything, they're useless. Or should server in this case provide straight linkage from `a` to `c`? Or am I missing something obviuos again?)

File: format/index.md [hunk 1]
```diff
@@ -955,10 +955,86 @@ COMMA, ",") list that refers to the name(s) of the fields to be returned.
...
+      }
+    }
+  ]
+}
+```
+
+Request with `fields` parameter:
+
 ```http
 GET /articles?include=author&fields[articles]=title,body&fields[people]=name
```
Comment (dgeb) [inline format/index.md:1003]: Relationships must also be included in `fields`, even if they're specified via `include`. Therefore, the `author` field would have to be included in `fields[articles]` to get the response below.

In other words, the request would need to be:

``` http
GET /articles?include=author&fields[articles]=title,body,author&fields[people]=name
```

**Comment by ksimka** [comment]
Thanks, @dgeb. Closing in favour of #691.

**Pull request closed without merging**